### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by mbdesigns1989**
To-Do list: interactive list

I used webpack to bundle JavaScript.
I Learnt how to use proper ES6 syntax.
I used ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,36 +1,67 @@
...
+  description: 'Complete To Do List project',
   completed: false,
   index: 2,
 }];
 
+function saveToStorage(taskArray) {
+  localStorage.setItem('tasks', JSON.stringify(taskArray));
+}
+
 function displayTasks() {
```
Comment (Emmyn5600) [inline src/index.js]: - [ ] Kindly, modify your function with `arrow function`.
arr1

File: src/status.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+function status(e, tasklist) {
+  if (e.target.classList.contains('unchecked')) {
+    e.target.classList.value = 'checked';
+    tasklist.completed = true;
+  } else if (e.target.classList.contains('checked')) {
+    e.target.classList.value = 'unchecked';
+    tasklist.completed = false;
+  }
+}
```
Comment (Emmyn5600) [inline src/status.js]: - [ ]  Kindly, modify your function with `arrow function`.
arr3

File: src/index.js [hunk 3]
```diff
@@ -1,36 +1,67 @@
...
+  }
+}
+
+function getFromStorage() {
+  const local = localStorage.getItem('tasks');
+  if (local) {
+    myTasks = JSON.parse(local);
+    displayTasks();
+  } else {
+    displayTasks();
```
Comment (Emmyn5600) [inline src/index.js]: - [ ] Kindly, modify your function with `arrow function`.
arr2

File: src/index.js [hunk 4]
```diff
@@ -1,36 +1,67 @@
...
-  completed: true,
+  description: 'Work on my social media',
+  completed: false,
   index: 1,
 },
 {
-  description: 'List structure project',
+  description: 'Complete To Do List project',
   completed: false,
   index: 2,
```
Comment (Emmyn5600) [inline src/index.js]: - [ ] You still have some features to be implemented.
arr4

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @mbdesigns1989 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Changes required :writing_hand: 


      * #### Please check the comments under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/index.js (hunk 4 above)
Comment (mbdesigns1989) [reply src/index.js]: Hello, I converted the functions to arrow functions but for the last request please check at http://localhost:8081 and check local storage in the browser it's working, other than that please explain, thanks.

Comment (Emmyn5600) [reply src/index.js]: The local Storage works perfectly, however, you need to add checkbox in the UI that make a user to change the status to true/false, and the task object's values for `completed` should be updated upon the user actions

Comment (mbdesigns1989) [reply src/index.js]: Hello, this is To-Do list: interactive list I already did everything as required, I asked my stand-up team and they said I don't need any changes.
Please tell me what do you mean by  "you need to add checkbox in the UI that makes a user to change the status to true/false, and the task's object's values for completed should be updated upon the user actions"
Please explain more I'm stuck here, thanks.

**Review by Emmyn5600 — APPROVED** [review]
Hi, 
Good job so far! 👏🏼 @mbdesigns1989 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 
 #### Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**